### PR TITLE
Gershom branch

### PR DESCRIPTION
**Pull request by gershomlapaix**
Hey guys! I would like to request you to pull and check my contribution if it is working in good way

### CODE REVIEW

File: src/com/customify/client/services/FeedbackService.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package com.customify.client.services;
+
+/**
+ * client feed back ser
```
Comment (veritem) [inline src/com/customify/client/services/FeedbackService.java:4]: Could you please addyour names to this class, date you created class as it was requested???

File: src/com/customify/client/views/CustomerFeedbackView.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package com.customify.client.views;
+
+/**
+ * This class file contains the view which allows the customer to provide his/her
```
Comment (veritem) [inline src/com/customify/client/views/CustomerFeedbackView.java:4]: Add you names please

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Add your names in methods and classes you defined.

File: src/com/customify/server/controllers/FeedbackController.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package com.customify.server.controllers;
+
+/**
+ * Controller for customer feedback
+ * */
```
Comment (veritem) [inline src/com/customify/server/controllers/FeedbackController.java:5]: Your names please

File: src/com/customify/server/services/FeebackService.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package com.customify.server.services;
+
```
Comment (veritem) [inline src/com/customify/server/services/FeebackService.java:2]: names please??

**Review by veritem — CHANGES_REQUESTED** [review]
Your names please!

File: src/com/customify/client/views/CustomerFeedbackView.java (hunk 2 above)
Comment (gershomlapaix) [reply src/com/customify/client/views/CustomerFeedbackView.java:4]: Let me add

**Pull request closed without merging**